[Metricbeat] Better document default metricsets (#12648)
elastic/beats: metricbeat/docs/modules/aerospike/namespace.asciidoc

include::../../../module/aerospike/namespace/_meta/docs.asciidoc[]

> This is a default metricset.

Reviewer: Will users know what we mean when we say that it is a default metricset? I wonder if it might be better to say something like "This metricset is enabled by default."

Author: Yah, I was debating about the language here. In this case, _default_ means "This metricset will be enabled if the module is not configured" which seems...subtly different? Maybe?

Reviewer: I think there's still some ambiguity though. Precise and concise are always at odds. :-)

Let's just leave it as you have it for now (This is a default metricset) and see if users are confused by the language.

Author: We could also link to the page that explains what a default module is too.

> This is a default metricset. If its host module is unconfigured, it will be enabled by default.

Reviewer 2: If it's not too late, I'd replace this text as follows. The reason is that the original is a bit pronoun heavy, which can make it harder to read.

```suggestion
This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.
```
